src/app/public/modules/file-attachment/file-attachment.component.scss

```diff
@@ -65,3 +66,53 @@
   height: 1px;
   overflow: hidden;
 }
+
+@include sky-theme-modern {
+  .sky-file-attachment-none {
+    font-size: 13px;
+  }
+
+  .sky-file-attachment-wrapper {
+    display: flex;
+    flex-wrap: wrap;
+
+    .sky-file-attachment-label-wrapper {
+      width: 100%;
+    }
+
+    .sky-file-attachment-preview-img {
+      @include sky-border(dark, top, bottom, left, right);
+      box-shadow: none;
+      margin: 0 10px 0 0;
+      width: 100px;
+      flex: 0;
+      order: 1;
+    }
+
+    .sky-file-attachment-btn {
+      margin: 0 10px 0 0;
```

Comment:
☝️ I believe we have SCSS variables for these values? `$sky-margin`

Reply:
Done.